kubeops/config-syncer-secret-generation-from-rancher-capi/01-policy.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+apply:
+- config-syncer-secret-generation-from-rancher-capi.yaml
```

Comment:
Ditto here with respect to comments made on the Argo policy above.

Reply:
Done!